src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -61,10 +67,29 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
     return undefined;
   }, [rest]);
 
+  const chatContext = useContext(ChatContext);
+
+  const submitMessage = useCallback(
+    async (data: { messageToTheBand: string }) => {
```

Comment:
I realise this is what `ChatBox` currently expects, but it feels somewhat needlessly complex/an implementation detail. I wonder if we can refactor this to just take a `msg: string` instead?